### ISSUE TITLE
Skywire-node does not kill therealsshapp/therealsshapp-client

### ISSUE THREAD

**Issue by ayuryshev** [skycoin/skywire-testnet#356]
# How to reproduce bug

1. Start 3 skywire-nodes: 
- NodeA with thereallssh app
- intermediary NodeB without apps
- NodeC with therealssh-client app
2. Add transports from NodeA and NodeC to NodeB
3. Start ssh communication. E.g. `therealssh-cli $PK_A "echo A"` (or `cat` a file, `ls`, `tree`, etc)
4. Kill nodes during proccess
5. Restart nodes, attempt to communicate, kill, repeat
6. Look in separate terminal: 
```bash
$ ps aux| grep "[t]hereal"
# isk      11149  0.3  0.2 846948 22024 pts/9    Sl+  12:49   0:00 /home/isk/github.com/skycoin/src/skywire/apps/therealssh-client.v1.0
# isk      12450  0.9  0.2 841096 16888 pts/9    Sl+  12:50   0:00 /home/isk/github.com/skycoin/src/skywire/apps/therealssh.v1.0
# isk      12917  1.9  0.2 840840 16740 pts/9    Sl+  12:51   0:00 /home/isk/github.com/skycoin/src/skywire/apps/therealssh.v1.0
# isk      13467  0.0  0.1 766332 11132 pts/9    Rl+  12:51   0:00 /home/isk/github.com/skycoin/src/skywire/apps/therealssh.v1.0
# isk      13477  0.0  0.1 766852 13268 pts/9    Rl+  12:51   0:00 /home/isk/github.com/skycoin/src/skywire/apps/therealssh-client.v1.0
```

Note multiple started `apps/therealssh.v1.0` and `apps/therealssh-client.v1.0`

Possibly related to #354 and #355

**Comment by Darkren** [comment]
Tried hard and couldn't reproduce the issue. Restarted nodes about 10 times, not hanging processes noticed

**Comment by jdknives** [comment]
Likely fixed by #403 . @ivcosla please confirm

**Comment by ivcosla** [comment]
Can confirm, they are being killed now

**Comment by jdknives** [comment]
Closed by #403
